guides/installation/devenv.md

```diff
@@ -370,6 +405,9 @@ Refer to the official devenv documentation to get a complete list of all availab
 
 ### Use customized VirtualHosts port for Caddy
 
+<Tabs>
+<Tab title="Port">
+
 ```nix
 // <PROJECT_ROOT>/devenv.local.nix
 { pkgs, config, lib, ... }:
```

Comment:
@shyim question for me was can we simply replace this with:

```
{
  services.caddy.virtualHosts.":8029" = {
    extraConfig = ''
      root * public
      php_fastcgi unix/${config.languages.php.fpm.pools.web.socket}
      file_server
    '';
  };
```

Reply:
sounds good.